convert_color_scheme.py

```diff
@@ -0,0 +1,135 @@
+"""Functionality for converting a color scheme to a "TerminalView" scheme."""
+import os
+import plistlib
+import sublime
+
+
+def hex_to_rgb(hexstring):
+    """Convert a string representing a hex color to an RGB tuple."""
+    return (int(hexstring[1:3], 16) / 255,
+            int(hexstring[3:5], 16) / 255,
+            int(hexstring[5:7], 16) / 255)
+
+
+def rgb_to_hex(rgb):
+    """Convert an RGB tuple to a hex string."""
+    return "#{:02x}{:02x}{:02x}".format(int(rgb[0] * 255),
+                                        int(rgb[1] * 255),
+                                        int(rgb[2] * 255))
+
+
+def norm2(vec3):
+    """Compute the squared norm of a three-dimensional vector."""
+    return vec3[0]**2 + vec3[1]**2 + vec3[2]**2
+
+
+def distance2(a, b):
+    """Compute the squared distance between two 3D vectors."""
+    return norm2((a[0] - b[0], a[1] - b[1], a[2] - b[2]))
```

Comment:
When I set the color reference for yellow to (255/255, 200/255, 0), it *does* choose the yellowish color. From some experimentation I noticed that the cie94 metric has some love/hate relationship with the green component. If you lower the green component in the reference value for yellow, there's almost no perceptual difference but the distances to things that look yellowish become much, much smaller.